packages/astro/src/core/endpoint/index.ts

```diff
@@ -12,7 +12,8 @@ import { ASTRO_VERSION } from '../constants.js';
 import { AstroCookies, attachCookiesToResponse } from '../cookies/index.js';
 import { AstroError, AstroErrorData } from '../errors/index.js';
 import { callMiddleware } from '../middleware/callMiddleware.js';
-import type { Environment, RenderContext } from '../render/index.js';
+import { type Environment, type RenderContext } from '../render/index.js';
+import { computePreferredLocale, computePreferredLocaleList } from '../render/context.js';
 
```

Comment:
`computePreferredLocales` seems unused

Reply:
`computePreferredLocale` is used at line 69

Reply:
Ah sorry my local branch of this was outdated.